config/_default/menus/main.en.yaml

```diff
@@ -4725,29 +4725,29 @@ menu:
       parent: llm_obs_instrumentation
       identifier: llm_obs_instrumentation_sdk
       weight: 204
-    - name: Monitoring
-      url: llm_observability/monitoring
+    - name: Exploration and Correlation
```

Comment:
let's keep the name of this root page to be "Monitoring". it's self-explanatory and fits in with hierarchy of the headers